### ISSUE TITLE
3.0.2 export 'undefined' (reexported as 'createFilter')

### ISSUE THREAD

**Issue by acajic** [JedWatson/react-select#3584]
when starting my react server, I get the following. 

```
Failed to compile.

./node_modules/react-select/dist/react-select.browser.esm.js
"export 'undefined' (reexported as 'createFilter') was not found in '../base/dist/react-select.browser.esm.js'

```

Recently I updated a lot of my packages, so this is the reason.

My `package.json`
```
{
  "name": "redspyclient",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@typescript-eslint/eslint-plugin": "^1.9.0",
    "acorn": "^6.1.1",
    "ajv": "^6.10.0",
    "autoprefixer": "9.5.1",
    "babel-core": "^6.26.3",
    "babel-eslint": "10.0.1",
    "babel-jest": "24.8.0",
    "babel-loader": "8.0.6",
    "babel-preset-react-app": "^9.0.0",
    "babel-runtime": "6.26.0",
    "case-sensitive-paths-webpack-plugin": "2.2.0",
    "chalk": "2.4.2",
    "css-loader": "^2.1.1",
    "dotenv": "8.0.0",
    "dotenv-expand": "5.1.0",
    "eslint": "5.16.0",
    "eslint-config-react-app": "^4.0.1",
    "eslint-loader": "2.1.2",
    "eslint-plugin-flowtype": "^2.50.3",
    "eslint-plugin-import": "2.17.3",
    "eslint-plugin-jsx-a11y": "6.2.1",
    "eslint-plugin-react": "7.13.0",
    "eslint-plugin-react-hooks": "^1.6.0",
    "file-loader": "3.0.1",
    "fs-extra": "8.0.1",
    "html-webpack-plugin": "^4.0.0-beta.5",
    "jest": "^24.8.0",
    "object-assign": "4.1.1",
    "postcss-flexbugs-fixes": "4.1.0",
    "postcss-loader": "3.0.0",
    "promise": "8.0.3",
    "raf": "3.4.1",
    "react": "^16.8.6",
    "react-cookie": "^4.0.0",
    "react-dev-utils": "^9.0.1",
    "react-dom": "^16.8.6",
    "react-router": "^5.0.0",
    "react-router-dom": "^5.0.0",
    "react-select": "^3.0.2",
    "resolve": "1.11.0",
    "style-loader": "0.23.1",
    "sw-precache-webpack-plugin": "0.11.5",
    "typescript": "^3.4.5",
    "url-loader": "1.1.2",
    "webpack": "^4.32.2",
    "webpack-dev-server": "^3.4.1",
    "webpack-manifest-plugin": "2.0.4",
    "whatwg-fetch": "3.0.0"
  },
  "scripts": {
    "start": "node scripts/start.js",
    "build": "node scripts/build.js",
    "test": "node scripts/test.js --env=jsdom"
  },
  "proxy": "http://localhost:3011",
  "jest": {
    "collectCoverageFrom": [
      "src/**/*.{js,jsx,mjs}"
    ],
    "setupFiles": [
      "<rootDir>/config/polyfills.js"
    ],
    "testMatch": [
      "<rootDir>/src/**/__tests__/**/*.{js,jsx,mjs}",
      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,mjs}"
    ],
    "testEnvironment": "node",
    "testURL": "http://localhost",
    "transform": {
      "^.+\\.(js|jsx|mjs)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|mjs|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs)$"
    ],
    "moduleNameMapper": {
      "^react-native$": "react-native-web"
    },
    "moduleFileExtensions": [
      "web.js",
      "js",
      "json",
      "web.jsx",
      "jsx",
      "node",
      "mjs"
    ]
  },
  "babel": {
    "presets": [
      "react-app"
    ]
  },
  "eslintConfig": {
    "extends": "react-app"
  }
}
```

What else can I submit to help get to the bottom of this?

**Comment by gwyneplaine** [comment]
@acajic i just released 3.0.3 that resolves this issue, thanks for the report

**Comment by gwyneplaine** [comment]
Closing this issue now @acajic
